### DISCUSSION TITLE
Suggestion : ark asa !online for all users

### DISCUSSION THREAD

**Discussion by Know1n3** [gameserverapp/Platform#802] [Suggestions]
Wanting to add a recommendation,  alot of my players like to see who is on from discord, currently using asa bot to do this, but know that GSA bot can do it with special admin permissions only, I'm sure a lot of us would rather only use 1 dashboard to manage there servers and GSA is almost a full asa bot replacements by bringing these features to us on gsa it would secure gsa as the better dashboard for asa, there isn't many !

**Comment by mvaessen** [comment]
thanks for the suggestion, this has been added
